app/dot/cpp/core/exceptions/EntityNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package dot.cpp.core.exceptions;
+
+public class EntityNotFoundException extends Exception {
+  public EntityNotFoundException() {
+    super("general.recordNotFound");
```

Comment:
translate its message to Romanian

Reply:
It was done in terra as there we store the messages